common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -14,10 +14,9 @@
 
 define(
 'video/01_initialize.js',
-['video/03_video_player.js', 'video/00_i18n.js'],
+['video/03_video_player.js', 'video/00_i18n.js', 'moment'],
 function (VideoPlayer, i18n) {
     var moment = window.moment;
```

Comment:
Thanks for the explanation. I think we just need to add a shim like we do for all the other vendor libraries (see https://github.com/edx/edx-platform/blob/master/lms/static/lms/js/require-config.js#L90):

            "moment": {
                exports: "moment"
            },

That should cause it ``window.moment`` to be returned as the value of the third parameter, and then code doesn't need to access it through ``window``. 

If that doesn't work, then just leave this as is.